### PR TITLE
PyCharm code check

### PR DESCRIPTION
**Pull request by ThomasWaldmann**
i just picked the easy stuff for now. but even here, it found a few real bugs.

thanks for qtile, I like it!

### CODE REVIEW

File: libqtile/log_utils.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def init_log(log_level=WARNING, log_path='~/.%s.log', log_truncate=False,
     # If we have a log path, we'll also setup a log file
     if log_path:
         try:
-            log_path = log_path % 'qtile'
+            log_path %= 'qtile'
```
Comment (flacjacket) [inline libqtile/log_utils.py]: Ooh, that's interesting, I didn't know string formatting could be a compound assignment operator

File: scripts/qtb [hunk 2]
```diff
@@ -33,8 +33,8 @@ def main():
             parser.error("Please specify widget name and argument.")
         try:
             client.widget.__getitem__(args[0]).update(" ".join(args[1:]))
-        except command.CommandError, v:
-            print >> sys.stderr, v
+        except command.CommandError as v:
+            print >>sys.stderr, v
```
Comment (flacjacket) [inline scripts/qtb]: Oh, interesting, something that got missed in the Python 3 move, I can take care of that seperately

**Comment by flacjacket** [comment]
Merged these commits, thanks for the much needed clean-up!

File: libqtile/log_utils.py (hunk 1 above)
Comment (ThomasWaldmann) [inline libqtile/log_utils.py]: Me neither. :)

**Pull request closed without merging**